src/App.tsx

```diff
@@ -1,52 +1,38 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
-
-// const tabs = [
-//   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-//   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-//   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-// ];
+import { Navigate, Route, Routes } from 'react-router-dom';
+import { TabsPage } from './components/TabsPage';
+import { PageNavLink } from './components/PageNavLink';
+import { HomePage } from './components/HomePage';
+import { NotFoundPage } from './components/NotFoundPage';
 
 export const App = () => (
   <>
-    {/* Also requires <html class="has-navbar-fixed-top"> */}
     <nav
       className="navbar is-light is-fixed-top is-mobile has-shadow"
       data-cy="Nav"
     >
       <div className="container">
         <div className="navbar-brand">
-          <a href="/" className="navbar-item is-active">Home</a>
-          <a href="/tabs" className="navbar-item">Tabs</a>
+          <PageNavLink to="/" text="Home" />
+          <PageNavLink to="tabs" text="Tabs" />
         </div>
       </div>
     </nav>
 
     <div className="section">
-      <div className="container">
```

Comment:
you delete some `container`
![image](https://github.com/mate-academy/react_tabs-with-router/assets/91411692/a722b333-90cf-423a-b9ad-
8b19a8ffc00c)